package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "bin": "bin.js",
   "dependencies": {
-    "airplay-protocol": "^2.0.2",
+    "airplay-protocol": "github:webtorrent/airplay-protocol",
```

Comment:
I thought we were trying to point this to `watson/airplay-protocol` since the fix is already on `master` there. Do we actually need a fork of `airplay-protocol`?